deploy/rbac-clowdapp.yml

```
@@ -271,37 +275,10 @@ objects:
           requests:
             cpu: ${CELERY_SCHEDULER_CPU_REQUEST}
             memory: ${CELERY_SCHEDULER_MEMORY_REQUEST}
-        env:
-        - name: DJANGO_LOG_LEVEL
-          value: ${DJANGO_LOG_LEVEL}
-        - name: DJANGO_DEBUG
-          value: ${DJANGO_DEBUG}
-        - name: APP_NAMESPACE
-          valueFrom:
-            fieldRef:
-              fieldPath: metadata.namespace
-        - name: PERMISSION_SEEDING_ENABLED
-          value: 'False'
-        - name: ROLE_SEEDING_ENABLED
-          value: 'False'
-        - name: GROUP_SEEDING_ENABLED
-          value: 'False'
-        - name: CLOWDER_ENABLED
-          value: ${CLOWDER_ENABLED}
-        - name: CW_NULL_WORKAROUND
-          value: ${CW_NULL_WORKAROUND}
-        - name: GLITCHTIP_DSN
-          valueFrom:
-            secretKeyRef:
-              name: ${GLITCHTIP_SECRET}
-              key: dsn
-              optional: true
-        - name: PRINCIPAL_USER_DOMAIN
-          value: ${PRINCIPAL_USER_DOMAIN}
+        env: 
+        - *common_env_vars
```

I think this should be just like `env: *common_env_vars`

Based on a quick test I did locally, I think this will embed the env vars array inside an outer array element. But we want the env vars array embedded directly.
```suggestion
        env: *common_env_vars
```

---

Oh, wait, you're trying to merge – I don't think that will work.